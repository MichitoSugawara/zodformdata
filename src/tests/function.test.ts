import { describe, expect, test, beforeAll } from "bun:test";
import sum from "../../dist";

describe("function", () => {
  test("add", () => {
    expect(sum(2,9)).toMatchSnapshot();
  });
});