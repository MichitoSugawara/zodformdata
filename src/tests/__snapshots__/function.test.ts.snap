// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`function add 1`] = `11`;
